oioioi/testrun/views.py

```diff
@@ -127,7 +127,10 @@ def download_input_file_view(request, submission_id):
     submission = get_submission_or_error(
         request, submission_id, TestRunProgramSubmission
     )
-    return stream_file(submission.input_file, name='input.in')
+    name='input.in'
+    if submission.input_is_zip:
+        name+='.zip'
```

Comment:
this would return "input.in.zip", what about using "input.in" for regular inputs and "input.zip" for zip files?

Reply:
My rationale was that that after unzipping the user should get the same filename as the one for uncompressed input, but now I see that the zipfile contains the input in a user-named file, so it's irrelevant.

While we're at it, would you agree that the filename should contain some additional information like it does for normal ins and outs (``problems_1_rok1a.in/.out``)?
I'd go with something like ``testruns_{submission_id}_{short_problem_name}.in/.zip/.out``.